bundle-workflow/src/test_workflow/perf_test_suite.py

```diff
@@ -0,0 +1,28 @@
+import os
+import subprocess
+
+
+class PerformanceTestSuite:
+    """
+    Represents a performance test suite. This class runs rally test on the deployed cluster with the provided IP.
+    """
+
+    def __init__(self, bundle_manifest, endpoint, security):
+        self.manifest = bundle_manifest
+        self.work_dir = 'tools/cdk/mensor/mensor_tests'
+        self.endpoint = endpoint
+        self.security = security
+        self.command = f'python3 test_config.py -i {self.endpoint} -b {self.manifest.build.id}'\
```

Comment:
So this test_config.py comes from Mensor? It needs to be doing dependency management with pipenv or similar, this is the wrong place to be calling `python3`.

Reply:
Yeah this file is present in the infra repo. I'll change the alias name, it should be `python`.

Reply:
No actually python3 is correct. But you see why this doesn't work well, it's precisely because this is setup specific. To make it setup-not-specific adopt pipenv in that tool, then this becomes `pipenv install` and `pipenv run ...`. All dependencies are under control and the environment is stable.

Reply:
Makes sense. Since I'm checking out to the infra repo, I've to install `pipenv` before running `pipenv install` https://github.com/opensearch-project/opensearch-build/pull/314/files#diff-207df8df9b6f387ef1a541037c5ed9ecda140695ec4c2ce16bdcbac0b41d260dR22